src/lib/generate-mdx-outline.ts

```diff
@@ -0,0 +1,40 @@
+import remark from 'remark'
+import { visit } from 'unist-util-visit'
+import generateSlug from '@hashicorp/remark-plugins/generate_slug'
+
+/** This function is used for the tutorial table of contents and also for search indexing */
+export interface OutlineItem {
+  slug: string
+  title: string
+}
+
+export default async function generateOutline(
+  mdxContent: string
+): Promise<OutlineItem[]> {
+  const headings = [{ title: 'Overview', slug: 'overview' }]
```

Comment:
In Learn we autogenerate an overview item that links to the tutorial metadata. I'm not sure that we want to do this in dev portal. My only hesitancy in removing it is if folks link to this already.